### ISSUE TITLE
Linux Mint (OEM Installation). No  pubkezy, no repositories. 

### ISSUE THREAD

**Issue by sunce21** [CISOfy/lynis#1447]
**Describe the bug**

**Version**
 - Distribution [e.g. Ubuntu 18.04]
 - Lynis version [e.g. 2.7.0]

**Expected behavior**

**Output**
If applicable, add output that you get from the tool or the related section of lynis.log

**Additional context**

**Comment by ariskemper** [comment]
@sunce21 this is probaby issue with wrong permission on cisofy-software-public.gpg. Set permission to 644.

**Comment by sunce21** [comment]
Aris, thank you for being helpful, and polite by answering me about the
subject. Strangely, nobody else did, and that skipped my logic. After the
first installation, all was good.
Just a day after, someone broke into my computer, and I had to install
everything from the beginning. Lost some crypto points on the Odyssey
platform also.
Clear theft 1/1, but what can I do... Support for Linux Mint is confusing.
Like "is this Ubuntu, Xubuntu, or Mint"? Updates are always for Ubuntu
L.T.S.
Now, I am not so interested in learning Python, Java or C++ in my fifties.
As a simple user O.S. should just work nice, secure, and stable.
Support kicked me for asking a simple question (or someone did something,
using my name). Thunderbird is unable to use my pgp key. Also, the product
key is missing. How can that be possible?
This project is strange to me. There is no support, or I am not able to
find one alone. When I was a teacher, I wanted my students to learn how to
mix using vinyls, in order to play fair.
Maybe not the best comparison, but it's about support. Thank you very much
once again,  being a good person is the most important thing.
Cheers,
Alex

**Comment by ariskemper** [comment]
@sunce21 no problem for help :), you could run command with sudo in priviliged mode like `sudo lynis audit system`, so then you will not have issue with other keys.

**Comment by sunce21** [comment]
:) Here is what I did, and I figured it out by myself (must admit, I saw
this in some conversation on Reddit)  sudo-add-apt-repository ppa:
Linerunner /tlp
After that, sudo-apt-install tlp tlp tlp. One part worked good, and still
does, but some parts were not installed (according to Terminal, which I
can' t trust as he fooled me before lol).
I will try to do what you told me, and we shall see the results.
I'm under contract, and I have to finish music for some incredibly boring,
hipster movie. Production is the field in which I move better.
If you want, send me some (irrelevant) email and I'll send you a good deep
house mix (exclusive).
I don't know how else to thank you? In any case, I highly appreciate the
fact that you came forward to help me.
The least I can do, and what I should do, is to send you some good deep
house music.
I'm not going back to Windows under any circumstances. If I have to study
at the age of fifty, I will study, but I'm not going back anyway.
Greetings from the whole team.
Alex
